src/views/ApartmentApp/components/StaffMenu/components/ApplicationTable/components/ApplicationRow/components/ApplicantSubTable/index.js

```diff
@@ -31,7 +31,7 @@ const ApplicantSubTable = ({ applicants }) => (
       <TableHead className={'stylized-table-head'}>
         <TableRow>
           <TableCell>Name</TableCell>
-          <TableCell align="center">Age</TableCell>
+          <TableCell align="center">Age (As of Sept. 1, 2020)</TableCell>
```

Comment:
```suggestion
          <TableCell align="center">Age (As of Sept. 1, 2021)</TableCell>
```

Reply:
If you could make this get the current year, that would be even better

Reply:
False, you should use the date object that already exists to do the date dynamically, so that a person doesn't have to go update the code by hand each year

Reply:
Also, did this come from Ethan, because I thought he told us that it is meant to be the previous September, not the upcoming September, based on the examples that he provided us